### PR TITLE
Recipe for adding libcerf

### PR DESCRIPTION
**Pull request by andyfaff**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by andyfaff** [comment]
@conda-forge/help-c-cpp

**Comment by andyfaff** [comment]
I asked @jwuttke directly if he would be listed as a maintainer.

File: recipes/cerf-feedstock/build.sh
Comment (xhochy) [inline recipes/cerf-feedstock/build.sh:3]:
```suggestion
cmake ${CMAKE_ARGS} ..
```

File: recipes/cerf-feedstock/build.sh
Comment (xhochy) [inline recipes/cerf-feedstock/build.sh:4]:
```suggestion
make -j$CPU_COUNT
```

File: recipes/cerf-feedstock/meta.yaml
Comment (xhochy) [inline recipes/cerf-feedstock/meta.yaml:1]: Please delete the comments.

File: recipes/cerf-feedstock/meta.yaml
Comment (xhochy) [inline recipes/cerf-feedstock/meta.yaml:37]: Please add some tests for the existence of relevant file, e.g. the shared object

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/cerf-feedstock/meta.yaml
Comment (andyfaff) [reply recipes/cerf-feedstock/meta.yaml:23]: I believe `test -f $CONDA_PREFIX/lib/libcerf.dylib` or `test -f $CONDA_PREFIX/lib/libcerf.so` should work

File: recipes/cerf-feedstock/meta.yaml
Comment (xhochy) [inline recipes/cerf-feedstock/meta.yaml:25]: In the build process, you should use `$PREFIX` instead. Also you can unify this as:
```suggestion
    - test -f "$PREFIX/lib/libcerf${SHLIB_EXT}"  # [unix]
```

Comment (andyfaff) [reply recipes/cerf-feedstock/meta.yaml:25]: I didn't know about those environment variables. I'll make the changes to the test section. Did you want me to make changes to `build.sh` as well?

Comment (xhochy) [reply recipes/cerf-feedstock/meta.yaml]: No, everything is fine now. We're only waiting for CI.

**Review by xhochy — APPROVED** [review]

**Comment by andyfaff** [comment]
@xhochy thank you.

**Pull request merged**
